source/_integrations/vacuum.xiaomi_miio.markdown

```diff
@@ -218,6 +218,17 @@ The following table shows the units of measurement for each attribute:
 
 ### Xiaomi Home app (Xiaomi Aqara Gateway, android?)
 
+<div class='note'>
+
+This method yielded an incorrect token on the 22nd of April, 2020 when used with:
+1. Xiaomi Gateway
+2. Xiaomi Mi Robot Vacuum
+3. Xiaomi IR Remote
+
+Using the alternate method with Mi Home `v5.4.49`, there were 3 different tokens listed for each of the above devices. The token yielded by this method matched the token for the Xiaomi Gateway in the text file. After using the correct token from the Mi Home `v5.4.49` method, the devices started to work again. 
+
+</div>
```

Comment:
- Should this be an alert box? Does this happen often that it should be highlighted that big?
- Should this be an alert on alerts.home-assistant.io instead?

I personally use this, but with the above text, I'm not even sure what I should do?

Reply:
1. Sure. An alert box sounds good.
2. Assuming this method works in some scenarios, then moving the alert box down would be fine too. 
3. Not sure what the alerts.home-assistant.io page does or what other processes it triggers so will leave that up to you. 

My intention is just to highlight to the rest of the community that this method isn't foolproof. 

New to github and first time contributing to open source. 
Please be gentle. :)